### ISSUE TITLE
r2 -H MAGICPATH is relative

### ISSUE THREAD

**Issue by radare** [radare/radare2#10758]
wtf

```
        R2_PREFIX=/usr/local
        MAGICPATH=share/radare2/2.8.0-git/magic
        PREFIX=/usr/local
        INCDIR=/usr/local/include/libr
        LIBDIR=/usr/local/lib
        LIBEXT=dylib
        RCONFIGHOME=/Users/xxx/.config/radare2
        RDATAHOME=/Users/xxx/.local/share/radare2
        RCACHEHOME=/Users/xxx/.cache/radare2
        LIBR_PLUGINS=/usr/local/lib/radare2/last
        USER_PLUGINS=/Users/xxx/.local/share/radare2/plugins
        USER_ZIGNS=/Users/xxx/.local/share/radare2/zigns
```

**Comment by gvanem** [comment]
Even worse on Windows. `radare2` seems unable to read/find my `%HOME%/.radara2rc`. And the colours *ass-u-me* a black background console. I.e. one of the worst ports to Windows I've seen in years.

**Comment by radare** [comment]
Wat

**Comment by ret2libc** [comment]
meson or acr?

**Comment by pelijah** [comment]
@gvanem because it uses `~/.config/radare2rc`

**Comment by pelijah** [comment]
Fixed in #10762
```
$ radare2 -H
R2_PREFIX=/usr/local
MAGICPATH=/usr/local/share/radare2/2.8.0-git/magic
PREFIX=/usr/local
INCDIR=/usr/local/include/libr
LIBDIR=/usr/local/lib/x86_64-linux-gnu
LIBEXT=so
RCONFIGHOME=/home/user/.config/radare2
RDATAHOME=/home/user/.local/share/radare2
RCACHEHOME=/home/user/.cache/radare2
LIBR_PLUGINS=/usr/local/lib/x86_64-linux-gnu/radare2/2.8.0-git
USER_PLUGINS=/home/user/.local/share/radare2/plugins
USER_ZIGNS=/home/user/.local/share/radare2/zigns
```
